habitat-baselines/habitat_baselines/utils/info_dict.py

```diff
@@ -23,11 +23,19 @@ def extract_scalars_from_info(info: Dict[str, Any]) -> Dict[str, float]:
     for k, v in info.items():
         if not isinstance(k, str) or k in NON_SCALAR_METRICS:
             continue
-        if k.split(".")[0] in NON_SCALAR_METRICS:
-            continue
+
+        if isinstance(v, dict):
+            result.update(
+                {
+                    k + "." + subk: subv
+                    for subk, subv in extract_scalars_from_info(v).items()
+                    if isinstance(subk, str)
+                    and k + "." + subk not in NON_SCALAR_METRICS
+                }
+            )
```

Comment:
I feel like this is reverting some changes that were made before. Is you commit history clean ?